integrations/electricity/barry.js

```diff
@@ -131,33 +155,45 @@ async function collect(state, { logWarning }) {
     We should probably use local time to define days
   */
 
-  const activities = Object.entries(groupBy(response, d => moment(d.date).startOf('day').toISOString()))
-    .map(([k, values]) => ({
-      id: `barry${k}`,
-      datetime: moment(k).toDate(),
-      activityType: ACTIVITY_TYPE_ELECTRICITY,
-      energyWattHours: values
-        .map(x => x.value * 1000.0) // kWh -> Wh
-        .reduce((a, b) => a + b, 0),
-      durationHours: values.length,
-      hourlyEnergyWattHours: values.map(x => x.value * 1000.0),
-      locationLon,
-      locationLat,
-    }));
+  const activities = Object.entries(
+    groupBy(response, (d) =>
+      moment(d.date)
+        .startOf('day')
+        .toISOString()
+    )
+  ).map(([k, values]) => ({
+    id: `barry${k}`,
+    datetime: moment(k).toDate(),
+    activityType: ACTIVITY_TYPE_ELECTRICITY,
+    energyWattHours: values
+      .map((x) => x.value * 1000.0) // kWh -> Wh
+      .reduce((a, b) => a + b, 0),
+    durationHours: values.length,
+    hourlyEnergyWattHours: values.map((x) => x.value * 1000.0),
+    locationLon,
+    locationLat,
+  }));
   activities
-    .filter(d => d.durationHours !== 24)
-    .forEach(d => logWarning(`Ignoring activity from ${d.datetime.toISOString()} with ${d.durationHours} hours instead of 24`));
+    .filter((d) => d.durationHours !== 24)
+    .forEach((d) =>
+      logWarning(
+        `Ignoring activity from ${d.datetime.toISOString()} with ${
+          d.durationHours
+        } hours instead of 24`
+      )
+    );
 
   if (!activities.length) {
     return { activities: [] };
   }
 
   // Subtract one day to make sure we always have a full day
   const lastFullyCollectedDay = moment(activities[activities.length - 1].datetime)
-    .subtract(1, 'day').toISOString();
+    .subtract(1, 'day')
+    .toISOString();
 
   return {
-    activities: activities.filter(d => d.durationHours === 24),
+    activities: activities.filter((d) => d.durationHours === 24),
```

Comment:
I think this rule breaks the linter no?

Reply:
Ah you're right, got confused since the `eslint-config-airbnb` [has since changed](https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb-base/CHANGELOG.md#1400--2019-08-09) to always use parentheses :)
I have update Prettier to use the same rule as our current config for consistency.